src/handlers/cell.ts

```diff
@@ -30,10 +30,10 @@ const CELL_CHANGED_TIMEOUT = 1000;
 export class CellManager implements IDisposable {
   constructor(options: CellManager.IOptions) {
     this._debuggerService = options.debuggerService;
+    this._id = options.debuggerService.session.client.name;
```

Comment:
Is this `_id` supposed to the unique? `client.name` can be the same for 2 notebooks (with different sessions) in 2 different folders (named `Untitled.ipynb` for example).

Reply:
Maybe we could create an id on the debug session.

Reply:
There is another `id` comparison happening in the notebook handler:

https://github.com/jupyterlab/debugger/blob/2c4a4ca854581aa9138686c7c6b9fc7defb71bd4/src/handlers/notebook.ts#L64

Maybe these two could be combined into one. Will look at it separately.